bull/pdl-queue/consumer.js

```diff
@@ -163,23 +171,32 @@ async function uploadZipToIA(zip, metadata, byteLength, email, job, onError) {
         headers: headers,
       },
       (error, response, body) => {
-        if (response.statusCode === 200) {
-          //EmailProducer(email, metadata.title, trueURI, true);
+        if (
+          response.statusCode === 200 &&
+          metadata.isEmailNotification === "true"
```

Comment:
What if `response.statusCode === 200` and  `metadata.isEmailNotification === "false"`? I think the whole nesting if else could be simplified.